make/check_kqueue.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ * InspIRCd -- Internet Relay Chat Daemon
+ *
```

Comment:
@SaberUK It's fine to add it in insp20 and it'll be added to master when insp20 is merged